WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsConnectView.swift

```diff
@@ -0,0 +1,31 @@
+import UIKit
+
+final class CardReaderSettingsConnectView: UIView {
```

Comment:
This looks bit more like a view controller than a view to me. I'm curious, what would be the benefit of declaring this as a view?